console-framework-client/src/main/java/io/axoniq/console/framework/client/SetupPayloadCreator.kt

```diff
@@ -66,7 +66,8 @@ class SetupPayloadCreator(
                 )
             },
             versions = versionInformation(),
-            upcasters = upcasters()
+            upcasters = upcasters(),
+            features = SupportedFeatures(true)
```

Comment:
Nit: can we use names for the parameter for clarity?
```suggestion
            features = SupportedFeatures(heartbeat = true)
```

Reply:
Added